src/platforms/javascript/common/sourcemaps/uploading/angular-webpack.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Angular CLI
+description: "Upload your source maps using the Angular CLI, a custom Angular builder and the Sentry Webpack plugin."
+sidebar_order: 0
+supported:
+  - javascript.angular
+---
+
+You can use the Angular CLI together with our Sentry Webpack plugin to automatically create releases and upload source maps to Sentry when building your app (e.g. with `ng build`).
+Due to Angular's closed build system, you first need to to configure a custom builder to register the Webpack plugin.
+
+## Installation
+
+```shell {tabTitle:npm}
+npm install --save-dev @angular-builders/custom-webpack @sentry/webpack-plugin
+```
+
+```shell {tabTitle:yarn}
+yarn add --dev @angular-builders/custom-webpack @sentry/webpack-plugin
+```
```

Comment:
```suggestion
## Install

Sentry captures data by using an SDK within your application’s runtime.

```shell {tabTitle:npm}
npm install --save-dev @angular-builders/custom-webpack @sentry/webpack-plugin
```

```shell {tabTitle:yarn}
yarn add --dev @angular-builders/custom-webpack @sentry/webpack-plugin
```
```

Reply:
Ah I agree, we should add a sentence here above the code section but we're not installing an SDK here but a webpack plugin. Will add another sentence.

Reply:
[f5d6de2](https://github.com/getsentry/sentry-docs/pull/6404/commits/f5d6de24350c97288998ee7a69ce8c5fbc596181)